lib/postprocess/image.js

```diff
@@ -25,6 +27,10 @@ const getDimension = value => {
  */
 const limit = pLimit(64)
 
+// load placeholder in case an image fails to load or is broken
+const placeholderSrc = fs.readFileSync(path.join(__dirname, 'placeholder.png')).toString('base64')
+const placeholder = Buffer.from(placeholderSrc, 'base64')

```

Comment:
I just realised that we are doing two reads - one into a regular buffer and then into a base64 encoded representation. consider reading to base64 directly.

https://stackoverflow.com/questions/28834835/readfile-in-base64-nodejs

Reply:
Reading from a file the way it is suggested in the SO post only returns a String in bas64, which we have to read into a Buffer anyways. I haven't found a way yet to do something short and direct like: Buffer.from(placeholder.png, base64) <- that's the step which takes the longest to execute

Reply:
The fasted way I've found is the one above: readFileSync returns a Buffer if no encoding is specified which then speeds up the conversion to base64 when reading into the placeholder Buffer. So we skip the conversion to String and back